analysis/node-wrapper/package.json

```diff
@@ -13,13 +13,15 @@
 	"version": "1.119.0",
 	"description": "MaibornWolff CodeCharta Analysis Tools",
 	"scripts": {
-		"prepare": "rm -rf ../build/install/codecharta-analysis public && ../gradlew -p .. installDist && cp -R ../build/install/codecharta-analysis public"
+		"prepare": "rm -rf ../build/install/codecharta-analysis public && ../gradlew -p .. installDist && cp -R ../build/install/codecharta-analysis public",
+		"postinstall": "node install.js"
 	},
 	"bin": {
 		"ccsh": "public/bin/ccsh"
 	},
 	"license": "BSD-3-Clause",
 	"dependencies": {
+		"is-windows": "^1.0.2",
```

Comment:
Please just check `process.platform === 'win32'`. The library is not maintained and not needed.